provisioning_templates/provision/autoyast_default.erb

```diff
@@ -22,21 +22,57 @@ oses:
   <networking>
     <dns>
       <hostname><%= @host.name %></hostname>
+      <% if @host.subnet.respond_to?(:dhcp_boot_mode?) -%>
+      <dhcp_hostname config:type="boolean">false</dhcp_hostname>
+      <dhcp_resolv config:type="boolean">false</dhcp_resolv>
+      <domain><%= @host.domain %></domain>
+      <% if @host.subnet.dns_primary.present? -%>
+      <nameservers config:type="list">
+        <nameserver><%= @host.subnet.dns_primary %></nameserver>
+        <% if @host.subnet.dns_secondary.present? -%>
+        <nameserver><%= @host.subnet.dns_secondary %></nameserver>
+        <% end -%>
+      </nameservers>
+      <% end -%>
+      <% else -%>
       <resolv_conf_policy>auto</resolv_conf_policy>
-<% if @host.domain -%>
+      <% end -%>
+    <% if @host.domain -%>
       <searchlist config:type="list">
         <search><%= @host.domain %></search>
       </searchlist>
-<% end -%>
+    <% end -%>
     </dns>
     <interfaces config:type="list">
       <interface>
-        <bootproto>dhcp</bootproto>
         <device>eth0</device>
+        <ethtool_options>autoneg on</ethtool_options>
+        <% if @host.subnet.respond_to?(:dhcp_boot_mode?) -%>
+          <bootproto>static</bootproto>
+          <ipaddr><%= @host.ip %></ipaddr>
+          <netmask><%= @host.subnet.mask %></netmask>
+          <network></network>
+          <startmode>onboot</startmode>
+        <% else -%>
+          <bootproto>dhcp</bootproto>
+        <% end -%>
         <startmode>auto</startmode>
         <usercontrol>no</usercontrol>
       </interface>
     </interfaces>
+    <% if @host.subnet.respond_to?(:dhcp_boot_mode?) -%>
+    <routing>
+    <ip_forward config:type="boolean">false</ip_forward>
+    <routes config:type="list">
+      <route>
+        <destination>default</destination>
+        <device>eth0</device>
+        <gateway><%= @host.subnet.gateway %></gateway>
+        <netmask>-</netmask>
```

Comment:
I don't know, but I find it all over the docs. https://www.suse.com/documentation/sles-12/book_autoyast/data/createprofile_network.html